core/execution/market.go

```diff
@@ -2182,7 +2146,7 @@ func (m *Market) zeroOutNetwork(ctx context.Context, parties []events.MarketPosi
 			Price:         settleOrder.Price.Clone(), // average price
 			OriginalPrice: settleOrder.OriginalPrice.Clone(),
 			CreatedAt:     now,
-			Reference:     ref,
+			Reference:     fmt.Sprintf("distressed-%d", i),
```

Comment:
maybe party is better than i for tracking, idk.

Reply:
Sounds good, happy to put it in place once there's a consensus we're ok to remove the random ID in the reference.